cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -35,20 +35,52 @@ To read more about the execution plan operators mentioned in this chapter, see <
 [[how-are-queries-executed]]
 == How are queries executed?
 
+_***XXTODO by Docs Team - change this heading to something like 'Preparser options' (this is what they are called internally), or something more reader-friendly (if a term other than 'preparser' is decided upon, please also change accordingly in the text). This section no longer deals with 'how queries are executed' per se. More of that content will be added by me later on. END TODO***_
```

Comment:
As suggested in more detail in the 3.1 review, I would consider 'Query execution engine selection'

Reply:
To the Docs Team: @mariascharin @jjaderberg @davidoliverSP2

Reply:
👍